indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -178,6 +181,7 @@ public RemoteTaskRunner(
     this.workerConfigRef = workerConfigRef;
     this.cleanupExec = MoreExecutors.listeningDecorator(cleanupExec);
     this.resourceManagement = resourceManagement;
+    this.runPendingTasksExec = Execs.multiThreaded(config.getPendingTasksRunnerNumThreads(), "rtr-pending-tasks-runner-%d");
```

Comment:
the comment "this method uses a single threaded executor" on `runPendingTasks` is no longer true, should be adjusted.